core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterDMTest.java

```diff
@@ -11032,6 +11033,43 @@ private RelNode createRelNodeWithQualifyStatement() {
   private static RelNode toLogical(RelNode rel, RelBuilder builder) {
     return rel.accept(new ToLogicalConverter(builder));
   }
+
+  @Test public void testSubQueryWithInClauseRule() {
+    final RelBuilder builder = relBuilder();
+    final RelNode subQueryInClause = builder.scan("DEPT")
+        .project(builder.field(0), builder.field(1))
```

Comment:
To make the relnode more readable.Please use builder.field("fieldname") instead of ordinal

Reply:
used builder.field("fieldname") instead of ordinal